contracts/WildcardSteward_v2.sol

```diff
@@ -47,6 +47,12 @@ contract WildcardSteward_v2 is Initializable {
     mapping(uint256 => uint256) public tokenGenerationRate; // we can reuse the patronage denominator
 
     MintManager_v2 public mintManager;
+    uint256 public auctionStartPrice;
```

Comment:
make mapping

Reply:
Sorry to bounce around, but I think it is still on the table that this value is fixed and the only time it is different is when the token just forclosed (then we take the tokens previous price).

Reply:
Lets do that and keep complexity low for now.

Reply:
Okay that makes sense, yeah happy with that. Edge case, the price when it forecloses, is less that our auction minimum 0.05. Will account for that